.github/workflows/issue-opened.yml

```
@@ -13,3 +13,4 @@ jobs:
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
         configuration-path: .github/labeler-issue-triage.yml
+        enable-versioned-regex: 1
```

Reviewing the [action documentation for this feature](https://github.com/github/issue-labeler#example-using-versioned-issue-templates), I do not believe this should be enabled.

```suggestion
        enable-versioned-regex: 0
```